src/can-map-attributes.js

```diff
@@ -1,17 +1,14 @@
-var can = require('can/util/util');
-require('can/map/map');
-require('can/map/map_helpers');
-require('can/list/list');
+var can = require('can-util');
+var Map = require('can-map');
+require('can-map/map-helpers');
+require('can-list');
 
 //!steal-remove-start
 can.dev.warn("can/map/attributes is a deprecated plugin and will be removed in a future release. "+
 	"can/map/define provides the same functionality in a more complete API.");
 //!steal-remove-end
 
-can.each([
-	can.Map,
-	can.Model
-], function (clss) {
+function applyAttributes (clss) {
```

Comment:
If this gets loaded before `can-model`, it will properly apply attributes features to the static Model class because it will already exist on Map at the time Model gets created.  Whereas merely importing `can-model` would just cause console errors (and that's really what I'm trying to avoid here).

The other option that I got from Matthew is to import can-util/namespace and see if can.Model exists already.  That would mean no steal trickery and the code would be simpler.